### PR TITLE
fix name of setpoint trace in feedforward debug

### PR DESCRIPTION
**Pull request by ctzsnooze**
Simple name change.

The setpoint trace in the feedforward debug currently shows the un-smoothed setpoint, for use when evaluating RC smoothing delays, and feedforward delays.

It isn't an interpolated setpoint trace, it is now an un-smoothed setpoint trace.

This only applies to firmware 4.3 (since a PR about 4 months ago), so no version control is needed.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
